include/openthread/netdata.h

```diff
@@ -274,6 +274,31 @@ uint8_t otNetDataGetVersion(otInstance *aInstance);
  */
 uint8_t otNetDataGetStableVersion(otInstance *aInstance);
 
+/**
+ * Check if the steering data allows a Joiner.
+ *
+ * @param[in]  aEui64             A pointer to the Joiner's IEEE EUI-64 or NULL for any Joiner.
+ *
+ * @retval OT_ERROR_NONE          The joiner is in the bloom filter.
+ * @retval OT_ERROR_NOT_FOUND     The joiner is not in the bloom filter.
+ *
+ */
+otError otNetDataSteeringDataHasJoiner(otInstance *aInstance, const otExtAddress *aEui64);
```

Comment:
```suggestion
bool otNetDataSteeringDataHasJoiner(otInstance *aInstance, const otExtAddress *aEui64);
```
Since we only return T/F, use bool?